docs/send-data/opentelemetry-collector/data-transformations.md

```diff
@@ -0,0 +1,112 @@
+---
+id: data-transformations
+title: OpenTelemetry - String Hashing and Masking using Transform Processor and OTTL
+sidebar_label: String Hashing and Masking
+description: Learn how to perform string hashing and masking operations using the Transform Processor and OTTL in OpenTelemetry.
+---
+
+OpenTelemetry provides the Transform Processor and OTTL (OpenTelemetry Transformation Language), empowering you to perform string hashing and masking operations on telemetry data. With the flexibility to configure the Transform Processor in your OpenTelemetry pipeline, you can replace sensitive information with hashed values or masked strings, ensuring data protection.
+
+You can find more detailed information about the available OTTL functions and their usage in the [OTTL Functions README](https://github.com/rnishtala-sumo/opentelemetry-collector-contrib/blob/ottl-replace-pattern/pkg/ottl/ottlfuncs/README.md).
+
+## Hashing examples
+
+### Example 1: Hashing a password in log body
+
+```yaml
```

Comment:
The inconsistent password regex between line 21 and 30 (`\w+`, `test`) had me a little confused.

Reply:
yeah one extracts the match group into an attribute called "password", the second one looks for the password in the message.

Maybe a better idea to change the second regex to something more generic like "password=([0-9A-Za-z]+_)"